core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/common/Status.java

```diff
@@ -12,7 +12,13 @@
     AVAILABLE(StatusKind.FINAL),
     UPDATE_IN_PROGRESS(StatusKind.PROGRESS),
     UPDATE_REQUESTED(StatusKind.PROGRESS),
+    BACKUP_REQUESTED(StatusKind.PROGRESS),
```

Comment:
This is a general comment for the review about how the status works on the cloudbreak side. It might or might not have any affect on what you do with status in your code. But on the cloudbreak side, there is no separate states of requested/started and in-progress. By the time the flow advances to the point where it's updating the stack status, the operation is already in progress. So in my version of these status messages, I had:
`
    BACKUP_IN_PROGRESS(StatusKind.PROGRESS),
    BACKUP_FINISHED(StatusKind.FINAL),
    BACKUP_FAILED(StatusKind.FINAL),
    RESTORE_IN_PROGRESS(StatusKind.PROGRESS),
    RESTORE_FINISHED(StatusKind.FINAL),
    RESTORE_FAILED(StatusKind.FINAL);
`
I see you have multiple places where you have a separate start state, and an in-progress state. For the datalake service I think that makes sense, since you can mark it as started as soon as you get the request, and in-progress when you get the flow identifier back from cloudbreak. I just wanted to give you a heads up that cloudbreak won't be using the REQUESTED stattus.

Reply:
I just followed the states defined for UPDATE. UPDATE has UPDATE_REQUESTED and UPDATE_IN_PROGRESS etc. 

I can update the patch accordingly.